libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/MapboxNavigationNotification.kt

```diff
@@ -264,29 +266,34 @@ internal class MapboxNavigationNotification : NavigationNotification {
     }
 
     private fun updateDistanceText(routeProgress: RouteProgress) {
-        if (currentDistanceText == null || newDistanceText(routeProgress)) {
-            currentDistanceText = distanceFormatter?.formatDistance(
-                routeProgress.currentLegProgress().currentStepProgress().distanceRemaining()
-            )
-            collapsedNotificationRemoteViews?.setTextViewText(
-                R.id.notificationDistanceText,
-                currentDistanceText
-            )
-            expandedNotificationRemoteViews?.setTextViewText(
-                R.id.notificationDistanceText,
-                currentDistanceText
-            )
+        ifNonNull(currentDistanceText, distanceFormatter) { distanceText, distanceFormatter ->
```

Comment:
Does we break the using ifNonNull for `currentDistanceText` ?? Because as you can see #267 line above, we had next condition:  `if (currentDistanceText == null` ...

Reply:
👍